scripts/run-e2e-tests.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# Run end-to-end tests
+#
+# Usage: ./scripts/run-e2e-tests.sh
+
+set -e
+
+SERVICE_HOST="${SERVICE_HOST:-host.docker.internal}"
+SERVICE_PORT="${SERVICE_PORT:-8080}"
+
+DOCKER_NETWORK_ARG="--network li-simulator-x1x2x3_default"
```

Comment:
Kann auf die Füße fallen je nach docker compose version. Ich würde den Network-Name dann in der compose.yml anpassen.